src/ceph-volume/ceph_volume/devices/lvm/listing.py

```diff
@@ -292,17 +226,13 @@ def main(self):
         )
 
         parser.add_argument(
-            'device',
-            metavar='DEVICE',
-            nargs='?',
+            'device', metavar='DEVICE', nargs='?',
             help='Path to an lv (as vg/lv) or to a device like /dev/sda1'
         )
 
         parser.add_argument(
-            '--format',
-            help='output format, defaults to "pretty"',
-            default='pretty',
-            choices=['json', 'pretty'],
+            '--format', help='output format, defaults to "pretty"',
```

Comment:
Please revert this change for readability.

Reply:
Done.